docroot/modules/custom/sitenow_dispatch/sitenow_dispatch.install

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * @file
+ * SiteNow Dispatch install hooks.
+ */
+
+/**
+ * Ensure sitenow_dispatch.settings config entity exists.
+ */
+function sitenow_dispatch_update_9001(&$sandbox) {
```

Comment:
Can't remember why this was new but the admin form probably was having issues if the config entity did not exist. This update might not be necessary if that admin form was tweaked.

Reply:
I think several settings weren't sticking on first save if the config didn't already exist, but that's been tweaked now.